oauth2_http/java/com/google/auth/http/AuthHttpConstants.java

```diff
@@ -0,0 +1,16 @@
+package com.google.auth.http;
```

Comment:
I could be okay with either. I'd probably assume having it correct in the initial commit for the new files in the PR. There seems to be little benefit in making the follow-up change larger. Your choice.